README.md

```diff
@@ -59,67 +27,83 @@ sudo apt-get update && sudo apt-get install ros-ROS_DISTRO-microstrain-inertial-
-   The source command may need to be run in each terminal prior to launching a ROS node.
 #### Launch the node and publish data
 The following command will launch the driver. Keep in mind each instance needs to be run in a separate terminal.
-            
-        roslaunch microstrain_inertial_driver microstrain.launch
-Optional launch parameters:
-- name: namespace the node will publish messages to, default: gx5
-- port: serial port name to connect to the device over, default: /dev/ttyACM0
-- baudrate: baud rate to open the connection with, default: 115200
-- imu_rate: sample rate for IMU data (hz), default: 100
-- debug: output debug info? default: false
-- diagnostics: output diagnostic info? default: true
-    
-To check published topics:
-        
-    rostopic list
+```bash
+roslaunch microstrain_inertial_driver microstrain.launch
+```
 
-**Example**: Connect to and publish data from two devices simultaneously  
-In two different terminals:
+The node has some optional launch parameters that can be specified from the command line in the format `param:=value`
+- `namespace` : namespace that the driver will run in. All services and publishers will be prepended with this, default: `/`
+- `node_name` : name of the driver, default: `microstrain_inertial_driver`
+- `debug`     : output debug logs, default: `false`
+- `params_file` : path to a parameter file to override the default parameters stored in [`params.yml`](./microstrain_inertial_driver/microstrain_inertial_driver_common/config/params.yml), default: empty
     
-    roslaunch microstrain_inertial_driver microstrain.launch name:=sensor1234
+#### Publish data from two devices simultaneously  
+
+1. Create the following files somewhere on your system (we will assume they are stored in the `~` directory):
+    1. `~/sensor_a_params.yml` with the contents:
+        ```yaml
+        port: /dev/ttyACM0
+        ```
+    2. `~/sensor_b_params.yml` with the contents:
+        ```yaml
+        port: /dev/ttyACM1
+        ```
+2. In two different terminals:
+    ```bash    
+    roslaunch microstrain_inertial_driver microstrain.launch node_name:=sensor_a_node namespace:=sensor_a params_file:="~/sensor_a_params.yml"
+    ```
+    ```bash    
+    roslaunch microstrain_inertial_driver microstrain.launch node_name:=sensor_b_node namespace:=sensor_b params_file:="~/sensor_b_params.yml"
+    ```
 
-    roslaunch microstrain_inertial_driver microstrain.launch name:=bestSensor port:=/dev/ttyACM1
 This will launch two nodes that publish data to different namespaces:
-- sensor1234, connected over port: /dev/ttyACM0
-- bestSensor, connected over port: /dev/ttyACM1
+- `/sensor_a`, connected over port: `/dev/ttyACM0`
+- `/sensor_b`, connected over port: `/dev/ttyACM1`
 
-An example subscriber node can be found here: [Microstrain Examples](./microstrain_inertial_examples)  
+An example subscriber node can be found in the [Microstrain Examples](./microstrain_inertial_examples)  
```

Comment:
capital S in MicroStrain. also probably doesn't need "the" but whatever you prefer